README.rst

```diff
@@ -60,6 +60,20 @@ against the supported Python versions.
 
   $ pipenv run tox
 
+Releasing a new version
```

Comment:
This is a subheader under the `Development` header currently, but maybe it should be it's own header? What do you think @davehunt ?